packages/analytics-js-plugins/__tests__/utilities/retryQueue/RetryQueue.test.ts

```diff
@@ -909,7 +909,8 @@ describe('Queue', () => {
     expect(batchQueue.batch).toEqual({
       enabled: true,
       maxSize: 1024,
-      maxItems: 1,
+      maxItems: '1',
```

Comment:
I am confused, are these strings instead of numbers?

Reply:
Yeah, I removed the data type coercion. The user has to pass sane values to the options.